doc/docs/modules/ROOT/pages/aura.adoc

```diff
@@ -11,12 +11,17 @@ This chapter describes considerations around using Neo4j Connector for Apache Sp
 
 link:https://neo4j.com/cloud/aura/[Neo4j Aura] is a fully managed database as a service providing Neo4j.
 
-== Remote Clients
+== Connecting to Aura
 
-Sometimes there will be remote applications that talk to Neo4j via official drivers, that want to use
-streams functionality.  Best practices in these cases are:
+Connecting to Neo4j Aura is similar to connecting to on-premises Neo4j, but keep in mind:
 
 * Always use a `neo4j+s://` driver URI when communicating with the cluster in the client application.  The optimal
 driver URI will be provided by Aura itself when you create a database
 * In Aura Enterprise consider creating a separate username/password for Spark access; avoid running all processes through the default
 `neo4j` account.
+
+== Connecting to Aura from Spark on Databricks
+
+Aura customers connecting from Databricks may encounter SSL handshake errors due to Databricks' custom Java security settings removing certain cipher support.
+
+See the Aura knowledge base article link:https://aura.support.neo4j.com/hc/en-us/articles/1500003161121[Connecting to Aura with Databricks] for more information and for steps to configure your Databricks cluster to support connections to Aura.
```

Comment:
Hi @Mogztter, on this PR we added a link to the new Aura KBase article on using Databricks with Aura via the Spark Connector, but the docs aren't building because `hyperlink` for some reason doesn't like that Aura's kbase is proxied by Cloudflare, which apparently is returning a 403 probably based on user-agent inspection (just a guess). Currently, we've added a skip pattern into `lint-links.js`, as a workaround, in order to get the docs built. Do you have any suggestion on how we can solve this?

Reply:
That's strange, 401 would make sense to ask for authentication but 403 is forbidden. Maybe Cloudfare is configured to deny automated requests.

> Currently, we've added a skip pattern into lint-links.js, as a workaround, in order to get the docs built. Do you have any suggestion on how we can solve this?

No, that's the right way to fix this issue :+1:

Reply:
Please note that `link:` is superfluous in this case. AsciiDoc will recognize that `https://aura.support.neo4j.com/hc/en-us/articles/1500003161121[Connecting to Aura with Databricks]` as a link.
Having said that, `link:` is harmless so you can keep it to make it more explicit.